docs/reference/indices/rollover-index.asciidoc

```diff
@@ -24,17 +56,102 @@ from the original (rolled-over) index.
 In this scenario, the write index will have its rollover alias' `is_write_index` set to `false`, while the newly created index
 will now have the rollover alias pointing to it as the write index with `is_write_index` as `true`.
 
-The available conditions are:
 
-[[index-rollover-conditions]]
-.`conditions` parameters
-[options="header"]
-|===
-| Name     | Description
-| max_age  | The maximum age of the index
-| max_docs | The maximum number of documents the index should contain. This does not add documents multiple times for replicas
-| max_size | The maximum estimated size of the primary shard of the index
-|===
+[[rollover-wait-active-shards]]
+===== Wait for active shards
+
+Because the rollover operation creates a new index to rollover to, the
+<<create-index-wait-for-active-shards,`wait_for_active_shards`>> setting on
+index creation applies to the rollover action as well.
+
```

Comment:
The "as well" isn't really necessary. (Total nit.)

Reply:
Fixed with 58898c608379c37ef519bfe6433c4e674a939ce9